### DISCUSSION TITLE
Welcome to ac-microcourses Discussions!

### DISCUSSION THREAD

**Discussion by sgbaird** [AccelerationConsortium/ac-microcourses#2] [Announcements]
## 👋 Welcome!

Please comment below with an introduction of yourself and describe you'd like to be involved in the AC's initiative to develop microcourses for self-driving labs. After taking a look at [the certificate and microcourse overviews](https://ac-microcourses.readthedocs.io/en/latest/), please provide your feedback in https://github.com/AccelerationConsortium/ac-microcourses/discussions/11.

I'll start out. My name is Sterling Baird (@sgbaird) and I direct the training programs at the Acceleration Consortium. Part of my role is to develop and deliver these microcourses.

**Comment by youmustfight** [comment]
My name is Mark Hansen, and I've been teaching myself bio/chem/materials science at a community lab in Brooklyn called Genspace (I've been making an [intro wet lab curriculum](https://undergroundgarden.club/) there, so I'm big fan of education efforts). Focused on atmospheric methane removal, which requires a lot of study of microbes and explorations in catalysts. Just got started this past week with ESP32 boards and coding some data streaming ideas!
